admin-frontend-settings.php

```diff
@@ -51,8 +51,8 @@ function jobman_display_conf() {
 						array( 'jobman_print_sort_box', 'jobman_print_template_box', 'jobman_print_app_settings_box', 'jobman_print_app_template_box' )
```

Comment:
Just looking for a Sanity Check beofre I Merge this to the Master.  Once you have checked Add a "NInja Star"  and we can then Merge .   Any questions  post em here and they should post to Slack